cmd/copy.go

```diff
@@ -638,6 +640,22 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 		cooked.preserveSMBInfo = false
 	}
 
+	if raw.preservePOSIXProperties == "" && areBothLocationsPOSIXAware(cooked.FromTo) {
+		if cooked.FromTo.IsUpload() {
+			raw.preservePOSIXProperties = common.EPosixPropertiesOption.FullFidelity().String()
+		} else if cooked.FromTo.IsDownload() {
+			raw.preservePOSIXProperties = common.EPosixPropertiesOption.SpecialFilesAndProperties().String() // todo: should we default to FullFidelity? It may not always work out well for the customer.
+		} // ignore BlobBlob, these properties persist via metadata & it doesn't make sense to explicitly ignore these bits of metadata.
+		// todo: should we consider appending it to a user's custom metadata if it's specified for BlobBlob?
```

Comment:
If this scenario is around "side-car'ing" POSIX metadata for uploads to Azure (flat/block) blobs, then yes that was the expectation that we'd leverage the x-ms-meta name:value header here -- One example workflow for eg: 

![image](https://user-images.githubusercontent.com/20822397/168382255-0a68be3d-007e-4f18-a76e-489766eb6a45.png)

Reply:
Directories, empty or filled, are persisted as an empty blob with the hdi_isfolder metadata.